### ISSUE TITLE
Questie turned off Halloween event 1 day early

### ISSUE THREAD

**Issue by Darkrenzo** [Questie/Questie#6647]
### Description

I was planning on spending the last day to do all the Halloween candy buckets and I log in to find that Questie is not showing them on the map anymore, even though the event doesn't end til 6pm.
This really shouldn't happen when the even is still running.

I am on the latest update as of 2 days ago and they were showing fine yesterday.

### Questie version

v10.6.4

### Game flavor

Cata

**Comment by Cabro** [comment]
The event has ended a few hours ago. At least that's what was shown in Calendar and what we noticed ingame.

**Comment by Darkrenzo** [comment]
Nope. Literally just did a bucket 60 seconds ago. Event doesn't end for 6 more hours.

**Comment by Darkrenzo** [comment]
Is there an option to manual reactivate the event map markers instead of questie auto hiding the quest markers?

**Comment by Darkrenzo** [comment]
![WoWScrnShot_110124_115549](https://github.com/user-attachments/assets/f9dd6b76-299d-4774-bdf2-326290266af7)

As you can see here, there is a bucket that can be done, there is no marker on the map for it, it also says that questie is auto hiding those markers, and the calendar says it ends at 6pm yet it is 12:50pm ST on the clock on the minimap.

**Comment by coldpasta123** [comment]
Looks like the event ends at a different time in the US vs EU (and not just for time difference...)
